app/views/citizens/income_summaries/_income_type_item.html.erb

```diff
@@ -0,0 +1,20 @@
+<li>
+  <h2 class="app-task-list__section" id="<%= "income-type-#{name}" %>">
+    <span class="app-task-list__section-number"><%= "#{number}. " %></span>
+    <%= t("transaction_types.names.#{name}") %>
+    <% if translation_exists_for?("transaction_types.hints.#{name}") %>
+      <span id="nationality-1-item-hint" class="govuk-hint" style="margin-bottom: 0px">
```

Comment:
I'd suggest another `id` :)
And because this id is not dynamic, it would appear several times in the page. Which is not valid
Does it even need an id?

Reply:
I've actually removed the id on this `span` it's not needed, at least at the moment.  And since I wrote this code, all the hints have been removed from the design, but am keeping it here because they'll probably come back